docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+networks:
+  api_server_default:
+    external: true
+services:
+  rollup-fullnode:
+    depends_on:
+      - geth
+    networks:
+      - api_server_default
+    build:
+      context: packages/rollup-full-node
+      dockerfile: Dockerfile
+    environment:
+      ENV_NAME: dev
```

Comment:
Should we just [pass through our local environment variables](https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers) so we can override in different environments?

Reply:
Ah, this actually runs without any environment variables set. I [removed that block for now](https://github.com/ethereum-optimism/optimism-monorepo/pull/39/commits/c53ec2446e659f2d155f5e1d16ae7fa8e0dd3cfd) and we can add it back in the follow-on PR if needed 👍